app/models/user.rb

```diff
@@ -3,4 +3,10 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  # AS OWNER
+  has_many :owned_donations, foreign_key: :owner_id, class_name: "Donation"
+
+  # AS CUSTOMER
+  has_many :calimed_bookings, foreign_key: :customer_id, class_name: "Donation"
```

Comment:
did you mean `claimed_bookings`?

Reply:
yes, This was fixed in recent Pull Request. I meant claimed_donations